views/include-file-upload.twig

```diff
@@ -29,6 +29,9 @@
         {{/if}}
         {{#if oldMediaId}}
             <input type="hidden" name="oldMediaId" value="{{ oldMediaId }}" />
+            <input name="folderId" type="hidden" id="folderId" value="{{oldFolderId}}">
```

Comment:
I'm not entirely sure, but this may need to be formFolderId (id), we may need to try that really and see

Reply:
OK good shout - lets try it and see.